kwant_to_openfermion.py

```diff
@@ -17,25 +67,37 @@ def system_to_FermionOperator(sys):
         The hamiltonian of sys as an openfermion object.
     '''
     
-    if not isinstance(sys, kwant.system.System):
-        raise TypeError(f'Expecting an instance of System, got {type(sys)}')
+    
+    #Get the number of spin states out of the first on-site value
+    sample_val = sys.onsites[0][0]
```

Comment:
Does kwant allow for the possibility of having different spins on each site? I.e. each 'val' that you pull from a Hamiltonian might not be square --- I can't remember if this is the case. But, if it is this wouldn't allow that, right?